src/components/CourseInfo/CourseInfo.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { mockedCoursesList, mockedAuthorsList } from 'src/constants';
+import getCourseDuration from 'src/helpers/getCourseDuration';
+
+interface ICourseInfo {
+	id?: string;
+	title?: string;
+	description?: string;
+	duration?: number;
+	authors?: string[];
+	creationDate?: string;
+}
+
+function CourseInfo() {
+	const { courseId } = useParams<{ courseId: string }>();
+	const course = mockedCoursesList.find((course) => course.id === courseId);
+	if (!course) {
+		return <div>Course not found</div>;
+	}
+
+	const authors = mockedAuthorsList.filter((author) => {
+		return course.authors.includes(author.id);
+	});
```

Comment:
suggestion: you can use implicit return here, it'll make the code easier to read